draft-irtf-cfrg-hpke.md

```diff
@@ -1131,16 +1189,18 @@ the PSK or AuthPSK mode, is proven in {{HPKEAnalysis}}; in a quantum
 setting, the remaining security level is smaller and defined by the
 post-quantum security level of the AEAD scheme.
 
-## Security Requirements on a KEM used within HPKE
+## Security Requirements on a KEM used within HPKE {#kem-security}
+
+The Encap/Decap interface of a KEM used within HPKE MUST be
+IND-CCA2-secure. An appropriate definition of this security notion for
+KEMs can be found in {{BHK09}}.
 
-A KEM used within HPKE MUST ensure the following to avoid identity
-mis-binding issues: The KEM shared secret computed by Encap and Decap MUST
-depend explicitly on the KEM public key `pkR` and the encapsulated key `enc`,
-as observed in {{S01}}. The KEM shared secret returned by AuthEncap and AuthDecap
-MUST explicitly depend on the KEM public keys `pkR` and `pkS` and the encapsulated
-key `enc`. This is usually implemented by including these values explicitly into
-the context of the key derivation function used to compute the KEM shared
-secret. This is also how DHKEM meets the requirement.
+The AuthEncap/AuthDecap interface of a KEM used within HPKE MUST be
+multi-user insider IND-CCA-secure and multi-user insider strongly
+unforgeable (sUF-CMA-secure) as defined in {{SigncryptionDZ10}}.
```

Comment:
To clarify: does DHKEM achieve this notion?

Reply:
DHKEM is not “multi-user insider strongly unforgeable (sUF-CMA-secure)” because of KCI.